consumers/messages.go

```diff
@@ -52,24 +63,56 @@ func handler(t transformers.Transformer, c Consumer) messaging.MessageHandler {
 	}
 }
 
-type filterConfig struct {
-	Filter []string `toml:"filter"`
+type subscriberConfig struct {
+	Subjects []string `toml:"subjects"`
+}
+
+type transformerConfig struct {
+	Format      string            `toml:"format"`
+	ContentType string            `toml:"content_type"`
+	Timestamps  []json.Timestamps `toml:"timestamps"`
 }
 
-type subjectsConfig struct {
-	Subjects filterConfig `toml:"subjects"`
+type config struct {
+	Subscriber  subscriberConfig  `toml:"subscriber"`
+	Transformer transformerConfig `toml:"transformer"`
```

Comment:
These fields are named ambiguously, maybe `SubscriberCfg` and `TransformerCfg` are better choices (since there are no fields that are actual Subscriber or Transformer in config).